packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx

```diff
@@ -0,0 +1,205 @@
+import * as React from 'react';
+import { IKeytipLayerProps, IKeytipLayerStyles, IKeytipLayerStyleProps } from './KeytipLayer.types';
+import { getLayerStyles } from './KeytipLayer.styles';
+import { Keytip, IKeytipProps } from '../../Keytip';
+import { Layer } from '../../Layer';
+import {
+  autobind,
+  BaseComponent,
+  KeyCodes,
+  KeytipTransitionModifier,
+  IKeytipTransitionKey,
+  ktpLayerId,
+  ktpAriaSeparator,
+  ktpAriaSeparatorId,
+  classNamesFunction
+} from '../../Utilities';
+import { KeytipManager } from '../../utilities/keytips';
+
+export interface IKeytipLayerState {
+  inKeytipMode: boolean;
+  keytips: IKeytipProps[];
+}
+
+const defaultStartSequence: IKeytipTransitionKey = {
+  key: 'Meta', modifierKeys: [KeytipTransitionModifier.alt]
+};
+
+const defaultExitSequence: IKeytipTransitionKey = {
+  key: 'Meta', modifierKeys: [KeytipTransitionModifier.alt]
+};
+
+const defaultReturnSequence: IKeytipTransitionKey = {
+  key: 'Escape'
+};
+const getClassNames = classNamesFunction<IKeytipLayerStyleProps, IKeytipLayerStyles>();
+
+/**
+ * A layer that holds all keytip items
+ *
+ * @export
+ * @class KeytipLayer
+ * @extends {BaseComponent<IKeytipLayerProps>}
+ */
+export class KeytipLayerBase extends BaseComponent<IKeytipLayerProps, IKeytipLayerState> {
+  public static defaultProps: IKeytipLayerProps = {
+    keytipStartSequences: [defaultStartSequence],
+    keytipExitSequences: [defaultExitSequence],
+    keytipReturnSequences: [defaultReturnSequence],
+    content: 'Alt Windows'
+  };
+
+  private _keytipManager: KeytipManager = KeytipManager.getInstance();
+  private _classNames: {[key in keyof IKeytipLayerStyles]: string };
```

Comment:
>_ [](start = 10, length = 1)

[nit] is the convention to use _ for private member variables only? If so, _ is missing in several of the example.tsx files